pkgs/applications/misc/dmrconfig/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub
+, libusb1, systemd }:
+
+stdenv.mkDerivation rec {
+  name = "dmrconfig-${version}";
+  version = "2018-10-20";
+
+  src = fetchFromGitHub {
+    owner = "sergev";
+    repo = "dmrconfig";
+    rev = "a4c5f893d2749727493427320c7f01768966ba51";
+    sha256 = "0h7hv6fv6v5g922nvgrb0w7hsqbhaw7xmdc6vydh2p3l7sp31vg2";
+  };
+
+  buildInputs = [
+    libusb1 systemd
+  ];
+
+  preConfigure = ''
+    substituteInPlace Makefile                                                 \
```

Comment:
nit: Why so much white space?

Reply:
I don't know, because it's pretty? :smile: 

I'll change it.